.gitignore

```diff
@@ -28,4 +28,7 @@ TAGS
 
 # Visual Studio Code
 .metals
-.bloop
\ No newline at end of file
+.bloop
+.vscode/settings.json
```

Comment:
let me know if these ignores are unwanted, they're just what I found in the diff after making some edits in vs code